swap/cheque.go

```diff
@@ -116,7 +116,7 @@ func (cheque *Cheque) verifyChequeProperties(p *Peer, expectedBeneficiary common
 		return fmt.Errorf("wrong cheque parameters: expected contract: %x, was: %x", p.contractAddress, cheque.Contract)
 	}
 
-	// the beneficiary is the owner of the counterparty swap contract
+	// the beneficiary is the issuer of cheques of the counterparty swap contract.
 	if err := cheque.VerifySig(p.beneficiary); err != nil {
```

Comment:
so if i understand this correctly, the mentioned `counterparty` here would be the peer `p`, right?

i find it confusing to call this `beneficiary`, as it could be interpreted as _the address this peer writes cheques to_ or _the address this peer wants its cheques written to_ just as well.

also: remove `.` from this comment

Reply:
Agree about `beneficiary` to be confusing. I would change it to `p.counterparty`